src/test/java/com/google/maps/DirectionsApiTest.java

```diff
@@ -394,4 +394,29 @@ public void testGeocodedWaypoints() throws Exception {
 
   }
 
+  /**
+   * Test {@code local_icon} for Directions in Paris.
+   */
+  @Test
+  public void testLocalIconInParis() throws Exception {
+    DirectionsResult result = DirectionsApi.newRequest(context)
+        .origin("paris metro bibliotheque francois mitterrand")
+        .destination("paris metro pyramides")
+        .mode(TravelMode.TRANSIT)
+        .await();
+    assertNotNull(result);
```

Comment:
I wouldn't even catch the NPE. JUnit will catch and report the error.

For debugging purposes would it be cleaner to just log the request/response? Or use a breakpoint?